lib/middleware.js

```diff
@@ -3,7 +3,7 @@ import { fileURLToPath } from 'node:url';
 import express from 'express';
 import swig from 'swig-templates';
 import fileUpload from 'express-fileupload';
-import swigFilters from './filters.js';
+import * as swigFilters from './filters.js';
```

Comment:
Oh! I didn't realize that `import * as <namespace> from 'string'` can be simplified to `import <namespace> from 'string'`. Maybe it's nicer to use the second one?

Reply:
To reduce bundle size is better to don't use default export.
`import <namespace> from 'string'` can only be used with default export (I removed it in that commit).